util/build_configs/cray-internal/setenv-xc-aarch64.bash

```diff
@@ -266,17 +266,17 @@ else
         list_loaded_modules
     fi
 
-    # Please keep the gen versions in compiler_versions.bash the same as these!
-    gen_version_gcc=7.3.0
-    gen_version_cce=9.1.3
+    # Please keep the target versions in compiler_versions.bash the same as these!
+    target_version_gcc=9.3.0
+    target_version_cce=10.0.2
```

Comment:
These are still the gen compilers, so we shouldn't change the name (just drop the comment)

Gen compilers are the compilers we use to build the module. Target compilers are the ones we use to test.